MetaMask/metamask-extension
# Update `@metamask/controllers` to v33

The controllers package has been updated to v33. The only breaking change in this release was to rename the term "collectible" to "NFT" wherever it appeared in the API.

Changes in this PR have been kept minimal; additional renaming can be done in separate PRs. This PR only updates the controller names, controller state, controller methods, and any direct references to these things. NFTs are still called "collectibles" in most places.

## Manual Testing Steps

To test the migration, you can setup a wallet using a build from `develop`, then switch to a build on this branch. You can compare the wallet state before and after to ensure the migration was correctly applied.

Aside from the migration, there are functional changes, so any standard regression testing steps would apply. Maybe pay special attention to any NFT-related functionality though.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
